packages/reactotron-app/App/Stores/SessionStore.js

```diff
@@ -10,6 +10,14 @@ const isSubscriptionCommandWithEmptyChanges = command => isSubscription(command)
 
 class Session {
 
+  // holds the list of commands to include in the list
+  @observable timelineCommandFilters = []
```

Comment:
@skellock this is the first use of an observable in the Session store. Not sure if you have a opinion on adding an observable here.